src/chrome/content/rules/Last.fm.xml

```diff
@@ -1,26 +1,31 @@
 <!--
 	For other CBS coverage, see CBS.xml.
-
 	For coverage of the last.fm API, see Audioscrobbler.com.xml.
 
-
 	CDN buckets:
+		- lastfm-img2.akamaized.net
+
+	Mismatch:
+		- m.cn.last.fm
+		- originals.last.fm
+		- trends.last.fm
+		- cdn.lst.fm
+	
+	Redirect:
```

Comment:
As long as they don't result in a redirect loop (redirect to HTTP) you should keep them as targets.